### DISCUSSION TITLE
Feedback for Xyarvius Pull request 01.08.2024

### DISCUSSION THREAD

**Discussion by 2kProjectEu4** [2kProjectEu4/Proper-2K-UI-Enhanced#2] [General]
Hi,
First thing first, thank you for the awesome changes that you made [Xyarvius](https://github.com/Xyarvius), I didn't expect so much stuff. 
I created this topic to discuss the changes I made after the Pull request. I added a google doc to explain why I temporary removed some stuff from the main branch, as I feel that I needed to explain my reason for the initial changes.

[Google Doc link](https://docs.google.com/document/d/16uve9FqPqI0dnAdSX00xh0_cEcaZdRfU/edit?usp=sharing&ouid=107896251154835177376&rtpof=true&sd=true)

I also extracted the terrain pictures that you added. I feel like this should be a separate mod, as some people may like the vanilla terrain pictures (for example :for me I like no know easily what to dev when i have extra powerpoints). 

I put the link below if you want to upload the mod yourself (if not I can do it).
[Download link](https://gofile.io/d/bQGQdO)

Thank you

**Comment by Xyarvius** [comment]
Hi Glidar,
nice, that sounds good. I will look at your comments soon. 

I am working an more changes, mostly small alignments here and there.

As for the terrain pictures: there are only 5 terrain pictures which overwrite vanilla terrain pictures and which thus would change the game without the corresponding mod. but i think it makes sense extracting them to a separate module anyways. It's probably cleaner that way.

**Comment by Xyarvius** [comment]
I agree with all of your comments 100%. I will make some changes accordingly.

**Comment by Xyarvius** [comment]
Here is a mockup of the diplomacy window, what do you think?

![image](https://github.com/user-attachments/assets/d2dfea11-8579-4001-bc5f-2d85149c2e64)

**Comment by Xyarvius** [comment]
Ok, i guess this means this is not to your liking? :D
Do you want to discuss these (and other changes) at all? I'd be fine with just moving these to a spearate module together with stuff like the mission frames and terrain pictures.
Or i can just put all my future changes there aswell if you'd rather do your own thing. Don't want to impose myself. :)

**Reply by 2kProjectEu4** [reply]
Sorry, I didn't have much time in the past 2 weeks (I was trying to finish 2 extension mods that I already started). 
Overall I like it as it keeps all the ruler/heir information together. The only thing I could not get used to was having the relations-related boxes not in the corner.
After some consideration, I decided to move the country shield up to free up space in the box. Here's how it looks now

![image](https://github.com/user-attachments/assets/ed28ead4-8afe-4649-9957-7544ce7c0a05)

**Reply by 2kProjectEu4** [reply]
About the other change for court view, I will try to look at it starting next week.

**Reply by Xyarvius** [reply]
Thanks for the reply. Sorry, didn't want to pressure.
Nice, I like this version. Did not feel quite right to me moving the opinion boxes,too. 
There are of course ruler names, which still not fit into this or any version with this box width, but most should.

**Comment by Xyarvius** [comment]
@2kProjectEu4 

Thanks for uploading the "Regional and urban terrain pictures" patch. 
I updated the files with a cleaner upscale:
https://drive.google.com/file/d/1wzlOhG_XjGRpDXvarMx4V8QyQF9Kvtn0/view?usp=sharing

I also made a upscaled minimap for myself some time ago. You can ahve a look if you prefer this version to the current one:
https://drive.google.com/file/d/1CNjecpv-6zFfWqiLuIC9g7JiuFefrhUK/view?usp=sharing

And i also made a patch for TGE. It 1:1 represents the current TGE but bigger (checked with winmerge). You could do a patch-submod too, if you want (still needs base TGE).
https://drive.google.com/file/d/1KbxnU1UGbdVYIyX8aRclmupITKUNjMFF/view?usp=sharing

**Reply by 2kProjectEu4** [reply]
@Xyarvius    Nice, I will take a look at them in the following days.

**Reply by Xyarvius** [reply]
@2kProjectEu4 Thanks.

I've also seen your discussion with Alzen in https://steamcommunity.com/sharedfiles/filedetails/?id=3320897976

I have upscaled the terrain pictures from https://steamcommunity.com/sharedfiles/filedetails/?id=2208088403
and also fixed the size of some of existing ones (in P2KE Patch - Regional and urban terrain pictures) directly in gfx/interface. You can add them somwhere, if you want to.
https://drive.google.com/file/d/1jeXvNIRqJRLToVgAr4A1HjiGFcxiAhco/view?usp=sharing

**Comment by 2kProjectEu4** [comment]
Hello @Xyarvius ,
I'm planning to delete some of the old branches. Could you let me know if you're still working on any of them so I don't accidentally remove something important?

Thanks!

**Reply by Xyarvius** [reply]
@2kProjectEu4 You can delete all the branches. :)

**Comment by Xyarvius** [comment]
@2kProjectEu4 I made a small update for the Platis Ages submod. I fixed the golden age banner position and fixed some formatting issues in the tooltips. You can download it here:
https://drive.google.com/file/d/1VSc4gJA7_aCmY4ssV29-ufbS8H27uS1E/view?usp=sharing
